### PR TITLE
doc: update known issues with peripheral_lbs sample bug

### PR DESCRIPTION
**Pull request by KAGA164**
Adds known bugs with tags to previous NCS versions.

Signed-off-by: Kamil Gawor <Kamil.Gawor@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -504,6 +504,12 @@ Subsystems
 Bluetooth LE
 ============
 
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-9820: The :ref:`peripheral_lbs` - while holding one of the other buttons, the notification for button release is the same as for button press.
```
Comment (peknis) [inline doc/nrf/known_issues.rst:509]: This is a bit unclear now when referring one of the other buttons. Which button release gives the same notification as the button press. I guess it is the button 1. We could perhaps use the description from the Jira ticket:
"If **button 1** pushed and released while holding one of the other buttons, the notification for release is the same as for press"

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -504,6 +504,12 @@ Subsystems
...
+
+  **Workaround:** Sample should filter which buttons were pressed before sending notification.
```
Comment (peknis) [inline doc/nrf/known_issues.rst:511]:
```suggestion
  **Workaround:** The sample should filter which buttons were pressed before sending a notification.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (joerchan) [inline doc/nrf/known_issues.rst:511]: Applications don't use samples, so no need to specify a workaround for the sample. Also modifying the code is not a workaround, it is the fix.

Comment (KAGA164) [reply doc/nrf/known_issues.rst]: Ok. I removed workaround section

Comment (KAGA164) [reply doc/nrf/known_issues.rst]: I removed this section

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (KAGA164) [reply doc/nrf/known_issues.rst]: Right. Changed

**Review by joerchan — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**